tsconfig.json

```diff
@@ -1,12 +1,18 @@
 {
   "compilerOptions": {
-    "target": "es5",
+    "allowSyntheticDefaultImports": true,
```

Comment:
Is this to let us import `connect` and `supertest`?

Reply:
Yes! 👍